clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -151,6 +152,21 @@
     public void resume(Collection<TopicPartition> partitions);
 
     /**
+     * @see KafkaConsumer#offsetsForTimes(java.util.Map)
+     */
+    public Map<TopicPartition, OffsetAndTimestamp> offsetsForTimes(Map<TopicPartition, Long> timestampsToSearch);
+
+    /**
+     * @see KafkaConsumer#earliestOffsets(java.util.Set)
+     */
+    public Map<TopicPartition, Long> earliestOffsets(Set<TopicPartition> partitions);
+
+    /**
+     * @see KafkaConsumer#latestOffsets(java.util.Set)
+     */
+    public Map<TopicPartition, Long> latestOffsets(Set<TopicPartition> partitions);
```

Comment:
We moved to using `Collection` for all other methods as part of KIP-45. Do we really want to use `Set` in these two methods?